CMakeLists.txt

```
@@ -238,6 +238,13 @@ set_target_properties(QCBOR PROPERTIES LINKER_LANGUAGE C)
 #        )
 target_link_libraries(infinisim PRIVATE QCBOR)
 
+if(EXISTS ${InfiniTime_DIR}/src/displayapp/fonts/CMakeLists.txt)
+  # available since https://github.com/InfiniTimeOrg/InfiniTime/pull/1097
+  message(STATUS "add subdirectory ${InfiniTime_DIR}/src/displayapp/fonts for 'infinitime_fonts' target")
+  add_subdirectory(${InfiniTime_DIR}/src/displayapp/fonts)
```

@Riksu9000 thanks for testing with an external InfiniTime repo. Could you try this modified `add_subdirectory` command. Currently don't have access to my PC

```suggestion
  add_subdirectory(${InfiniTime_DIR}/src/displayapp/fonts fonts)
```

---

That seems to work 👍